Public/Src/Engine/UnitTests/Scheduler/PipFingerprinterTests.cs

```diff
@@ -400,6 +401,94 @@ public void TestUserProfileStaysOutOfFingerprints()
             XAssert.IsTrue(json.Contains(Path.GetFileName(Environment.GetFolderPath(Environment.SpecialFolder.InternetCache)).ToUpperInvariant()));
         }
 
+        private ContentFingerprint GetWeakFingerPrintForExecutable(
+            string exectuablePath, 
+            bool assertValue,  // true if fingerprint should contain executable path, false if output should be untracked
+            bool untracked = false, 
+            string untrackedScopePath = null)
+        {
+            var pathTable = m_context.PathTable;
+            var executable = FileArtifact.CreateSourceFile(AbsolutePath.Create(pathTable, X(exectuablePath)));
+
+            var dependencies = new HashSet<FileArtifact>() { executable };
+            var processBuilder = GetDefaultProcessBuilder(pathTable, executable)
+                .WithDependencies(dependencies);
+
+            if (untracked)
+            {
+                processBuilder = processBuilder.WithUntrackedPaths(new AbsolutePath[] { AbsolutePath.Create(pathTable, X(exectuablePath)) });
+            }
+
+            if (untrackedScopePath != null)
+            {
+                processBuilder = processBuilder.WithUntrackedScopes(new AbsolutePath[] { AbsolutePath.Create(pathTable, OperatingSystemHelper.IsUnixOS ? "/tmp" : Environment.GetFolderPath(Environment.SpecialFolder.History)) });

```

Comment:
In addition to untracking `/tmp` you should untrack `untrackedScopePath` as well